### ISSUE TITLE
Indent guide improved

### ISSUE THREAD

**Issue by smlombardi** [Briles/gruvbox-atom#6]
Any thoughts on styling the colors use by the[ indent-guide-improved package](https://atom.io/packages/indent-guide-improved) to better mesh with the Gruvbox palette?

``` less
.indent-guide-improved {
  background-color: gray;
  &.indent-guide-stack {
    background-color: cyan;
    &.indent-guide-active {
      background-color: blue;
    }
  }
}
```

**Comment by michaeldeboeve** [comment]
this is how I did it

```
@neutral-red: #cc241d;
@neutral-green: #98971a;
@neutral-yellow: #d79921;
@neutral-blue: #458588;
@neutral-purple: #b16286;
@neutral-aqua: #689d6a;
@neutral-orange: #d65d0e;

@guide: @neutral-blue;

.indent-guide-improved {
  background-color: @guide;
  opacity: .1;
  &.indent-guide-stack {
    background-color: @guide;
    opacity: .3;
    &.indent-guide-active {
      background-color: @guide;
      opacity: .6;
    }
  }
}
```

**Comment by smlombardi** [comment]
Nice.  Thanks.

**Comment by Briles** [comment]
Thanks for bringing this to my attention. I haven't added support for many 3rd-party packages but if people come to me with a package they want supported, I'll gladly support it.

The changes I made in 8672d253bdbc0efcdd7c1e15e80ec3505e91ac75 should make it into the next release sometime this week.

**Comment by smlombardi** [comment]
When is this going to be released?

**Comment by Briles** [comment]
@smlombardi I just published v1.2.5 which includes these changes
